Dockerfile

```diff
@@ -20,8 +20,8 @@ ENV PROJECT_ROOT /app/
 ENV DJANGO_SETTINGS_MODULE=who.settings.docker \
     CELERY_APP=who
 
-# RUN LANGUAGE_CODE=en SECRET_KEY=compilemessages-key django-admin compilemessages && \
-RUN SECRET_KEY=collectstatic-key django-admin collectstatic --noinput && \
+RUN LANGUAGE_CODE=en SECRET_KEY=compilemessages-key django-admin compilemessages && \
+    SECRET_KEY=collectstatic-key django-admin collectstatic --noinput && \
     SECRET_KEY=compress-key django-admin compress
```

Comment:
@KaitCrawford Is this repeat assignment to the same constant correct? Otherwise 👍 if so

Reply:
yeah. It's how we do it for gem as well. It's just setting a secret key because django will complain if you try to run a command without one set. It doesn't actually use it